packages/react/package.json

```diff
@@ -41,41 +41,39 @@
     "test": "jest --silent",
     "test:watch": "jest --watch"
   },
-  "devDependencies": {
+  "dependencies": {
     "@pixi/react-components": "*",
     "@pixi/react-fiber": "*",
     "@pixi/react-invariant": "*",
-    "@pixi/react-modular": "*",
+    "@pixi/react-modular": "*"
```

Comment:
These are bundled into the `@pixi/react` dist as a single package so we don't need them as dependencies do we?

Reply:
I think I have misunderstood again how rollup is working in this project

Do we need to bundle everything into this package?

I feel like this should work more like pixi where the main `pixi.js` package is just a wrapper for a whole bunch of `dependencies`

Reply:
Perhaps not, I can try it without bundling here, I guess I was just trying to emulate how the original package worked in spitting out a single build file and anyone that needed to configure things separately still could by importing the individual modules